docs/chat_setup/slack_setup.md

```diff
@@ -0,0 +1,131 @@
+   - There should now be a "Bot User OAuth Access Token" displayed, typically a string starting with `xoxb-`.
+     You will need to configure this value for the plugin as the `slack_api_token` value, either directly or through an
+     `.env` file. If this value is not properly configured, the bot will be unable to send content to the user.
+6. Returning to the "Basic Information" page for your app, under "Display Information", you can specify the name,
+   description, icon, and accent/background color for the app. You can use the `nautobot_logo.png` from this
+   directory if desired.
+7. Proceed to the [Nautobot Server Preparation and Configuration](./chat_setup.md#Nautobot Server Preparation and Configuration) section.
+
+> **Note**: In the Slack app, you will need to invite the chatbot to each channel that it will belong to with `@<app name>`.
+> For example, when an app named `Nautobot Chatops` is installed to the workspace:
+> 1. A message is displayed in the channel, saying that the integration`Nautobot Chatops` has been added
+> 2. You `@Nautobot Chatops` in the channel
+> 3. You are prompted to add `@Nautobot Chatops` to the channel
+
+![slack integration invite](../images/add_nautobot.png)
+
+ # Configuring Multiple Chatbots in a Workspace
+
+Chatbots from multilple Nautobot implementations can exist in a single Slack workspace and even channel.
+
+They will be differentiated in the workspace using the `slack_slash_command_prefix` value in `PLUGINS_CONFIG`.
+
+Here is an example `nautobot_config.py` for the first Nautobot chatbot implementation in the workspace. 
+This chatbot will be called in the workspace using `/nautobot`.
+```python
+# Enable installed plugins. Add the name of each plugin to the list.
+PLUGINS = ["nautobot_chatops"]
+
+PLUGINS_CONFIG = {
+    'nautobot_chatops': {
+        'enable_slack': True,
+        'slack_api_token': '<slack-api-token>',
+        'slack_signing_secret': '<slack-signing-secret>',
+    }
+}
+```
+
+Here is an example `nautobot_config.py` for a second Nautobot chatbot implementation in the workspace.
+This configuration explicitly configures the `slack_slash_command_prefix` key/value.
+This chatbot will be called in the workspace using `/network2-nautobot`.
+
+```python
+# Enable installed plugins. Add the name of each plugin to the list.
+PLUGINS = ["nautobot_chatops"]
+
+PLUGINS_CONFIG = {
+    'nautobot_chatops': {
+        'enable_slack': True,
+        'slack_api_token': '<slack-api-token>',
+        'slack_signing_secret': '<slack-signing-secret>',
+        'slack_slash_command_prefix': '/network2-'
+    }
+}
+```
+
+> NOTE: by default, your slash command must end with `nautobot`. If no `slack_slash_command_prefix` is specified,
+> the slash command will be `/nautobot`. If a `slack_slash_command_prefix` is specified, the slash command will be `<slack_slash_command_prefix>nautobot`.
+
+> NOTE: Custom chatbot development allows for chatbot slash commands such as `/grafana` and `/meraki`.
```

Comment:
```suggestion
> NOTE: Custom chatbot development allows for chatbot slash commands such as `/grafana` and `/meraki`.

[Chat Setup](./chat_setup.md)
```

Adding a small bit of navigation to help with the User experience if having to go back and forth.